modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -610,6 +610,18 @@ var UICheckbox = UIElement.extend(/** @lends LuCI.ui.Checkbox.prototype */ {
 
 		frameEl.appendChild(E('label', { 'for': id }));
 
+		var tooltip = this.options.tooltip(this.options.name);
+		if (typeof(tooltip) === 'string') {
```

Comment:
I'd rather test for `!== false` here. The `E()` function allows other, non-string values as content too, e.g. an array of child nodes or a DOM Node element which could've been returned by a user defined `tooltip()` implementation.

Reply:
I'm not sure yet if this works with the changes I've made.